src/land/ghcn_snod2ioda.py

```diff
@@ -146,29 +158,32 @@ def assignValue(colrowValue, df400):
             lons = lons[mask]
             lats = lats[mask]
             alts = alts[mask]
+            sites = sites[mask]
             times = times[mask]
 
         # get datetime from input
         my_date = datetime.strptime(startdate, "%Y%m%d")
         start_datetime = my_date.strftime('%Y-%m-%d')
-        base_datetime = start_datetime + 'T12:00:00Z'
+        base_datetime = start_datetime + 'T18:00:00Z'
         AttrData['date_time_string'] = base_datetime
 
         for i in range(len(vals)):
```

Comment:
Why multiply a value by one and place it back into the same variable?  Just skip the line completely.
Can the setting of err to the number 40 happen using a simple if-statement like this ```errs = 40 if vals >= 0```
I don't know python well enough to know for sure it operates on all elements of a list using an if-statement like this. If yes, you can eliminate the for loop and do a single line assignment.

Reply:
The original code is to convert mm into m. Now the user requested to use mm rather than m. I have removed that sentence to avoid the confusion. Yes, it works for me.  The code cannot do a single line assignment (it does not work)  as I have to use a loop from my experience.

Reply:
@Greg, I believe that I addressed all your comments. If your time is available, please see if you have any further comments. Otherwise, you see if you approve this PR.